modules/runner/src/main/ts/core/Utils.ts

```diff
@@ -0,0 +1,22 @@
+export const noop = (): void => {};
+
+export const makeUrl = (session: string, offset: number, failed: number, retry: number): string => {
+  const baseUrl = window.location.protocol + '//' + window.location.host + window.location.pathname;
+  if (offset > 0) {
+    const rt = (retry > 0 ? '&retry=' + retry : '');
+    return baseUrl + '?session=' + session + '&offset=' + offset + '&failed=' + failed + rt;
+  } else {
+    return baseUrl;
+  }
+};
+
+export const formatElapsedTime = (start: Date, end: Date): string => {
```

Comment:
This code reminded me of `leftpad`, and the whole problem it caused. So I was curious to see what its current state was: https://www.npmjs.com/package/left-pad

It claims it's been superseded by https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart 

Do we trust that function? It looks like it isn't supported in IE? With the new separation of the parts of bedrock, I'm assuming `runner` is something that would run on the actual browser so it would need to work for IE?

Reply:
Yeah, runner needs to be able to work on IE 11 still.